packages/EasyErrorHandler/src/Bridge/Symfony/Builders/ApiPlatformBuilderProvider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Builders;
```

Comment:
We use singular namespaces in Symfony, so this should be `EonX\EasyErrorHandler\Bridge\Symfony\Builder`.

Reply:
Fixed.